screens/HomeScreen.js

```diff
@@ -22,7 +22,12 @@ class HomeScreen extends React.Component {
 
   render() {
     return (
-      <Text>Loading...</Text>
+      <View>
```

Comment:
would be good to have Loading as a component and just import it for all our loading screens. 

ps: there might even actually be better ways to abstract this away so all you have to do is given a parent component the prop 'isLoading', but that's not important right now